include/sound/hdaudio_ext.h

```diff
@@ -27,8 +27,8 @@ void snd_hdac_ext_stream_spbcap_enable(struct hdac_bus *chip,
 				 bool enable, int index);
 
 int snd_hdac_ext_bus_get_ml_capabilities(struct hdac_bus *bus);
-struct hdac_ext_link *snd_hdac_ext_bus_link_at(struct hdac_bus *bus, int addr);
-struct hdac_ext_link *snd_hdac_ext_bus_get_link(struct hdac_bus *bus,
+struct hdac_ext_link *snd_hdac_ext_bus_to_hlink_at(struct hdac_bus *bus, int addr);
+struct hdac_ext_link *snd_hdac_ext_bus_to_hlink(struct hdac_bus *bus,
 						const char *codec_name);
```

Comment:
I think the second function name change is not helping... Based on the name `snd_hdac_ext_bus_to_hlink` I would expect that it would 'convert' the hdac_ext_bus to hlink.
`snd_hdac_ext_bus_get_hlink_by_name()` perhaps or without the by_name postfix?

Reply:
Good suggestion, we can use by_name() and by_address(). Thanks!

Reply:
at_address instead maybe?